apidocs/openapi/users.yaml

```diff
@@ -1391,14 +1391,6 @@ components:
           operationId: getUser
           parameters:
             userID: $response.body#/id
-        get_groups:
-          operationId: listUsersInGroup
-          parameters:
-            groupID: $response.body#/id
-        get_channels:
-          operationId: listUsersInChannel
-          parameters:
-            channelID: $response.body#/id
```

Comment:
Just to clarify, Schemathesis still uses Open API links for stateful testing. However, the `--stateful` CLI option was removed

Reply:
These are links to operations that were deprecated, it seems they  have only been caught now